wfe2/wfe.go

```diff
@@ -1103,7 +1085,6 @@ func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *request
 	id := request.URL.Path
 	authz, err := wfe.SA.GetAuthorization(ctx, id)
 	if err != nil {
-		logEvent.AddError("No such authorization at id %s", id)
 		// TODO(#1199): handle db errors
```

Comment:
I think this TODO can be deleted (Maybe not a concern of this PR)

Reply:
This is actually still a live issue - distinguishing 404s from 500s.